src/utils/navigation.ts

```diff
@@ -39,6 +39,16 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.SECURE_UPDATE_FILTER_URL,
     nextPage: [config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE]
   },
+  [config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL]: {
+    currentPage: config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE,
+    previousPage: () => config.OVERSEAS_ENTITY_QUERY_PAGE,
+    nextPage: [config.WHO_IS_MAKING_UPDATE_PAGE]
+  },
+  [config.WHO_IS_MAKING_UPDATE_URL]: {
+    currentPage: config.WHO_IS_MAKING_UPDATE_PAGE,
+    previousPage: () => config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE,
+    nextPage: [config.OVERSEAS_ENTITY_REVIEW_PAGE]
```

Comment:
This will also change with follow on PRs:
nextPage: [config.UPODATE_DUE_DILIGENCE_URL, config.UPDATE_OVERSEAS_ENTITY_DUE_DILIGENCE_URL]